src/templates/contacts.html.j2

```diff
@@ -0,0 +1,64 @@
+{% extends "base.html.j2" %}
+{% block title -%}
+OSG Contacts
+{%- endblock %}
+{% block content %}
+{% if not authorized %}
+<div class="alert alert-info" role="alert">
+  To see contact information, load your grid certificate into your browser.
+</div>
+{% endif %}
+<div class="alert alert-info" role="alert">
+  Profile information is provided by the user and may be out of date.
+</div>
+<div class="container">
+  {%- for user in users %}
+    <div class="row text-white bg-dark">
+      <div class="col-sm"><span class="font-weight-bold">{{ user.FullName }}</span></div>
+      <div class="col-sm-auto text-right">
+        <span class="font-weight-bold">ID:</span>
+        <span class="text-monospace">{{ user.ID }}</span>
+      </div>
+    </div>
+    <div class="row">
+      <p class="col-sm-8">
+        {% if user.Profile -%}
+        {{ user.Profile|replace("\n", "<br>") }}
+        <br>
+        {%- else -%}
+        <span class="text-danger">(no profile)</span>
+        {%- endif %}
+      </p>
+    </div>
+    <div class="row">
+      {% if user.GitHub -%}
+      <div class="col-sm-3"><span class="font-weight-bold">GitHub:</span></div>
+      <div class="col-sm"><a href="https://github.com/{{user.GitHub}}">{{ user.GitHub }}</a></div>
+      {%- endif %}
+    </div>
+    {% if authorized -%}
+    <div class="row">
+      <div class="col-sm-3"><span class="font-weight-bold">Primary Email:</span></div>
+      <div class="col-sm"><a href="mailto:{{user.ContactInformation.PrimaryEmail}}">{{ user.ContactInformation.PrimaryEmail }}</a></div>
+    </div>
+      {% if user.ContactInformation.SecondaryEmail %}
+      <div class="row">
+        <div class="col-sm-3"><span class="font-weight-bold">Secondary Email:</span></div>
+        <div class="col-sm"><a href="mailto:{{user.ContactInformation.SecondaryEmail}}">{{ user.ContactInformation.SecondaryEmail }}</a></div>
+      </div>
+      {% endif %}
+      {% for lbl, attr in [
+        ("Primary Phone", "PrimaryPhone"),
+        ("Secondary Phone", "SecondaryPhone"),
+        ("IM", "IM")] %}
+      {% if user.ContactInformation[attr] %}
+      <div class="row">
+        <div class="col-sm-3"><span class="font-weight-bold">{{ lbl }}:</span></div>
+        <div class="col-sm">{{ user.ContactInformation[attr] }}</div>
+      </div>
+      {% endif %}
+      {% endfor %}
+    {%- endif %}
+  {% endfor -%}
+</div>
+{% endblock %}
```

Comment:
No newline??

Reply:
Geez you'd really think Git would have told me about that.